recipe/bld_win.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+cd $SRC_DIR || exit 1
+
+# Inspired by PKGBUILD
+export HOST=${MINGW_CHOST:-x86_64-w64-mingw32}
+export BUILD=${HOST}
+
+export CFLAGS="${CFLAGS} -Wno-attributes -Wno-ignored-attributes"
+
+# Build shared libraries.
+# To be aware: 'make' can build only one library at a time: shared or static but not both.
+mkdir -p "shared" && cd "shared"
+
+../configure --prefix="${PREFIX}" --host="${HOST}" --enable-cxx --enable-fat --enable-shared --disable-static
+
+make -j${CPU_COUNT}
+make check -j${CPU_COUNT}
+make install
+
+# Copy import libraries
+cp "${PREFIX}"/lib/libgmp.dll.a "${PREFIX}"/lib/libgmp.lib
```

Comment:
In the `configure` script, I see
```
    # Incidentally, libtool does generate an import library libgmp.dll.a,
    # but it's "ar" format and cannot be used by the MS linker.  There
    # doesn't seem to be any GNU tool for generating or converting to .lib.
```

Reply:
OK, _if_ we think we need MS import libraries then maybe we need to run the equivalent steps in https://github.com/AnacondaRecipes/intel_repack-feedstock/blob/main/recipe/dll2lib.bat -- although that then might require the MS compilers to get `dumpbin`

But, before we jump on that bandwagon, would an MS import library pointing at a GCC DLL even work?

I would be tempted to not rename these files and remove the section until some circumstances requires MS import libraries.  We're only building `gmp` for Windows for R and R uses GCC throughout.

Reply:
Agree. We can remove the section but then we won't test it in run_test.bat with cl.exe

Reply:
Removed